tensorly/decomposition/tests/test_parafac2.py

```diff
@@ -472,3 +473,47 @@ def test_parafac2_to_tensor():
         Bi = T.dot(projections[i], factors[1])
         manual_tensor = T.einsum("r,jr,kr", factors[0][i], Bi, factors[2])
         assert_(tl.max(tl.abs(constructed_tensor[i, :, :] - manual_tensor)) < 1e-6)
+
+
+def test_update_imputed():
+
+    rng = tl.check_random_state(1234)
+
+    random_parafac2_tensor = random_parafac2(
+        shapes=[(15, 30) for _ in range(25)],
+        rank=3,
+        random_state=rng,
+    )
+
+    # test mode='factors' imputation
```

Comment:
I think it would be easier to understand if this comment (and the comment on line 503) was a complete sentence explaining what the test is checking

Reply:
Thanks for the input @MarieRoald! This was addressed in bcb0fe51b06f07d48ab37217f17f7d148495796e.